SmartDeviceLink/SDLButtonName.h

```diff
@@ -20,6 +20,11 @@ typedef SDLEnum SDLButtonName SDL_SWIFT_ENUM;
  */
 extern SDLButtonName const SDLButtonNameOk;
```

Comment:
`SDLButtonNameOk` documentation should be updated. Not sure on what versions **Ok** started to be equivalent to **Play/Pause** but that information should be added. Also add that starting with SDL Core version 5.0, it will be the same as sending a `SDLButtonNamePlayPause`.

Reply:
Done.